README.md

```diff
@@ -18,28 +15,23 @@ Are you a Developer using the OCI SDK? If so, please fill out our survey to help
 
 
 ## Installing
-OCI Go SDK supports both `go get` and Go Modules installing.
-
 If you want to install the SDK under $GOPATH, you can use `go get` to retrieve the SDK:
 ```
 go get -u github.com/oracle/oci-go-sdk
 ```
-Alternatively, you can also clone the repo from [Github](https://github.com/oracle/oci-go-sdk).
-
-
-If you are using Go modules, you can install the latest or specific version(v25.0.0+) when adding the following in the go.mod file:
-```go
-require github.com/oracle/oci-go-sdk/{major-version} {version}
+If you are using Go modules, you can install by running the following command within a folder containing a `go.mod` file:
 ```
-One example is:
-```go
-require github.com/oracle/oci-go-sdk/v38 v38.0.0
+go get -d github.com/oracle/oci-go-sdk/@latest
+```
+Alternatively, you can install a specific version(supported from `v25.0.0` on):
+```
```

Comment:
```go
require github.com/oracle/oci-go-sdk/{major-version} {version} 
```had better be there.

Reply:
If you browse the `go` language issue tracker much, you'll see that you're not really supposed to worry about managing `go.mod` directly, but instead use `go get` and `go mod` to manage it.